mozilla-frontend-infra/firefox-health-dashboard
# Issue #83 - Add error handling

Fix for #83 
@sarah-clements I went with your suggestion using error panel to display error as I'm not comfortable with HOC yet I would try that v soon when I feel confident working with that. 

To test:
go to http://localhost:5000/quantum/64 and check for failed telemetry requests, an error message is displayed here. 

Disconnect from the internet, go to http://localhost:5000/quantum/tp6?bits=64 
You will see error msgs when there is no data. 

For this case, having the wrong parameter ( 323 instead of 32 of 64) 
https://health.graphics/quantum/323 

The code is written to handle this issue as 
if the user enters '32' then show data related to '32' bit else show '64' bit data no matter what value is entered. [See code](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/quantum/index.jsx#L36-L42)

So if the user enters 32 then it will render 32 bit data otherwise for any case it will throw 64 bit. 

Only this case is left: https://health.graphics/quantum/tp6?bits=63 
which I am figuring out to throw error its in process

Meanwhile please review the current PR.
